tcms/report/data.py

```diff
@@ -616,7 +664,7 @@ def _get_report_data_with_builds(self, form, builds):
 
         # Get related tested_by's username. Don't need duplicated user ids.
         tested_by_ids = []
-        for build_id, tested_bys in status_matrix.items():
+        for _, tested_bys in status_matrix.items():
```

Comment:
FYI prefix build_id with _. This will keep the information what the first element is but tell pylint we don't care about it.